test/next_ls/extensions/elixir_extension/code_action/unused_variable_test.exs

```diff
@@ -9,14 +9,18 @@ defmodule NextLS.ElixirExtension.UnusedVariableTest do
   alias NextLS.ElixirExtension.CodeAction.UnusedVariable
 
   test "adds an underscore to unused variables" do
-    text = """
-    defmodule Test.Unused do
-      def hello() do
-        foo = 3
-        :world
-      end
-    end
-    """
+    text =
```

Comment:
We get the text as a list of lines, which for the unused variable code isn't a problem but it's better to be consistent if anyone copies simple tests from other files like me.